src/code/code_800EC960.c

```diff
@@ -4607,22 +4610,22 @@ void Audio_PlaySceneSequence(u16 seqId) {
             // Start the sequence from the beginning
 
             // Writes to ioPort 7. See `SEQ_FLAG_SKIP_HARP_INTRO` for writing a value of 1 to ioPort 7.
-            skipHarpIntro = (sSeqFlags[seqId & 0xFF & 0xFF] & SEQ_FLAG_SKIP_HARP_INTRO) ? 1 : (u8)SEQ_IO_VAL_NONE;
+            skipHarpIntro = (SEQ_FLAGS(SEQ_ID(seqId)) & SEQ_FLAG_SKIP_HARP_INTRO) ? 1 : (u8)SEQ_IO_VAL_NONE;
             Audio_PlaySequenceWithSeqPlayerIO(SEQ_PLAYER_BGM_MAIN, seqId, 0, 7, skipHarpIntro);
 
             if (!(sSeqFlags[seqId] & SEQ_FLAG_RESUME_PREV)) {
                 // Reset the sequence resumePoint
                 sSeqResumePoint = SEQ_RESUME_POINT_NONE;
             }
         }
-        sPrevSceneSeqId = seqId & 0xFF;
+        sPrevSceneSeqId = SEQ_ID(seqId);
     }
 }
 
 void Audio_UpdateSceneSequenceResumePoint(void) {
     u16 seqId = Audio_GetActiveSeqId(SEQ_PLAYER_BGM_MAIN);
 
-    if ((seqId != NA_BGM_DISABLED) && (sSeqFlags[(u8)seqId & 0xFF] & SEQ_FLAG_RESUME)) {
+    if ((seqId != NA_BGM_DISABLED) && (SEQ_FLAGS(SEQ_ID(seqId)) & SEQ_FLAG_RESUME)) {
```

Comment:
in general with this macro it feels a bit weird that youre putting in a seq id and getting out a seq id. does the packed var need a more generalized name?

Reply:
Yeah, it does actually but when I started digging into it, I realized it's a bit of a rabbit whole and deserves a separate PR. 

But yes, it should be important to go through and distinguish all uses of what's a `packed` sfxId vs. what contains just the `sfxId` itself. Tempted to just remove this macro from this PR and address it in a followup PR

Reply:
separating it is fine with me